### DISCUSSION TITLE
Rotor Thrust

### DISCUSSION THREAD

**Discussion by phildsouza** [OpenFAST/openfast#1644] [General]
Dear @jjonkman ,

https://forums.nrel.gov/t/aerodyn-outputs/392/16
based on the above discussion I understand that rotor thrust is Elastodyn output which is given by LSShftFxa or RotThrust. But in elastodyn file I cannot locate the same. Is it because the output has been moved to some other module or I need to switch on some flag to generate rotor thrust.
openfast version v3.4.1.

**Comment by jjonkman** [comment]
Dear @phildsouza,

Are outputs `LSShftFxa` and/or `RotThrust` included in the `OutList` of the ElastoDyn input file you are using?   Only the outputs entered in the `OutList` will be written to the output file(s).

FYI: The full list of output parameters available from each module is documented here: https://openfast.readthedocs.io/en/main/_downloads/3f19498a5dc774461e022b671ff01ec6/OutListParameters.xlsx.

Best regards,
